extensions/cloud/portability-cloud-google/src/test/java/org/datatransferproject/cloud/google/GoogleCloudIdempotentImportExecutorTest.java

```diff
@@ -0,0 +1,84 @@
+package org.datatransferproject.cloud.google;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.Transaction;
+import com.google.cloud.datastore.testing.LocalDatastoreHelper;
+import java.io.IOException;
+import java.util.UUID;
+import jdk.nashorn.internal.scripts.JO;
+import org.datatransferproject.api.launcher.Monitor;
+import org.datatransferproject.types.transfer.errors.ErrorDetail;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class GoogleCloudIdempotentImportExecutorTest {
+
+  private static final String ITEM_NAME = "item1";
+  private static final Monitor monitor = Mockito.mock(Monitor.class);
+
+  private static final UUID JOB_ID = UUID.randomUUID();
```

Comment:
I wonder if we should initialize this is the setUp method in case we ever want to have more job id's. Or we can wait for the time that happens.

Reply:
I added a test to make sure we dont load any items from a different JOB_ID, but I think we can still keep it as final static since this test is pretty straightforward. If we want to expand later we can